anchore_engine/services/policy_engine/engine/feeds/feeds.py

```diff
@@ -461,21 +531,29 @@ def sync(
         ...
         }
 
-        :param: fetched_data
-        :param: full_flush
-        :param: event_client
-        :param: operation_id
-        :return: changed data updated in the sync as a list of records
+        :param fetched_data: disk cache for downloaded data to sync
+        :type fetched_data: LocalFeedDataRepo
+        :param full_flush: whether or not to flush all records before syncing new ones
+        :type full_flush: bool, defaults to False
+        :param event_client: catalog client
+        :type event_client: Optional[CatalogClient], defaults to None
+        :param operation_id: UUID4 hexadeicmal string
+        :type operation_id: Optional[str], defaults to None
+        :param group: filter for which groups to update
+        :type group: Optional[str], defaults to None
+        :return: FeedSyncResult, containing sync results for each group
+        :rtype: FeedSyncResult
         """
-        result = build_feed_sync_results()
-        result["feed"] = self.__feed_name__
+        result = FeedSyncResult(feed=self.__feed_name__)
         failed_count = 0
 
         # Each group update is a unique session and can roll itself back.
         t = time.time()
```

Comment:
Nit: I know you did not add this, but it might be nice to rename the single char variable here.

Reply:
done